charts/ceph-csi-cephfs/templates/provisioner-clusterrole.yaml

```diff
@@ -15,7 +15,7 @@ rules:
     verbs: ["get", "list"]
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "create", "delete","patch"]
+    verbs: ["get", "list", "watch", "create", "update", "delete","patch"]
```

Comment:
same RBAC is not required for rbd? :thinking:

Reply:
rbd has this RBAC already.